odpf/stencil/v1beta1/stencil.proto

```diff
@@ -306,20 +306,27 @@ message DeleteVersionResponse {
   string message = 1;
 }
 
-message SearchSchemasRequest {
+message SearchRequest {
   string namespace_id = 1;
-  int32 version_id = 2;
-  string query = 3;
+  string schema_id = 2;
+  int32 version_id = 3;
+  string query = 4;
+  bool all = 5;
```

Comment:
@ramey What is the role of all here?

Reply:
`all` returns matches from all the versions, by default only the matches from the latest versions are returned

Reply:
Make sense. Is there some better name we can think of for this? At first look I got confused that it is related to pagination,

Reply:
how does `scan_all_versions` sound?

Reply:
what we need is oneof between `version_id` and `all` fields. User can't set both fields together right? If they do, which field should take precendence?

Reply:
If both are specified then `version_id` will get the precedence.

Reply:
Hmm, we can think of using oneof to make it more explicit. 
Also what if we use latest as param instead of all. which by default is true? latest makes it more explicit? WDYT?

Reply:
The problem is while processing the value in the application. In case no value has been specified as it is a `bool` it defaults to `false` and there is no way to find out whether it is explicitly given by the user or default `bool` value. 
Agreed with `oneof`, will change that.

Reply:
make sense. In that case, another name of all could be history as well.

Reply:
@ramey @harikrishnakanchi lets go with history then?

Reply:
@ravisuhag I am planning to go with `history`

Reply:
@ramey cool.